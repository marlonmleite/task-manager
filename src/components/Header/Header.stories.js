import React from 'react'
import { storiesOf } from '@storybook/react'
import Header from './'

const storie = storiesOf('Header', module)

storie.add('default', () => (
  <div style={{ height: 200 }}>
    <Header />
  </div>
))
