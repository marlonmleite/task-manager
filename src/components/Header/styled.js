import styled from 'styled-components'

const Header = styled.nav`
  background-color: #f5f5f5;
  height: 36px;
  padding: 10px;
  box-shadow: 0 0px 12px 0px #0006;
`

export default Header
