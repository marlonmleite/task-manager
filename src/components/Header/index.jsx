import PropTypes from 'prop-types'
import React, { PureComponent } from 'react'
import Logo from 'core/assets/svg/geru.svg'
import { Header as HeaderStyled, Title } from './styled'

class Header extends PureComponent {

  static propTypes = {
    title: PropTypes.string.isRequired,
  }

  render() {
    const { title } = this.props

    return (
      <HeaderStyled>
        <Logo />
        <Title>{title}</Title>
      </HeaderStyled>
    )
  }

}

export default Header
