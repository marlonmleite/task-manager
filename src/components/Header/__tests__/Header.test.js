import React from 'react'
import { shallow } from 'enzyme'
import Header from '../'

describe('Header', () => {
  it('should render', () => {
    const header = shallow(<Header />)

    expect(header).toMatchSnapshot()
  })
})
