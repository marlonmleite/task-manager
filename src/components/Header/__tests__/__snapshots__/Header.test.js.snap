// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<styled.nav>
  <jest-stub />
  <styled.h1 />
</styled.nav>
`;
